### ISSUE TITLE
Moumts are down if CSI restart happens  

### ISSUE THREAD

**Issue by nikiladonya** [ctrox/csi-s3#34]
Wnem CSI  container mounts buckets it launchs mount daemons (when execute s3fs mount command for example) 
But if container died daemons in container died also and mounts will be failed. 
What can be done with this?

**Comment by ctrox** [comment]
Yeah I'm aware of this. I'm looking for a workaround but this might be pretty difficult to solve as all the mounting happens in user space (fuse) and because of the stateless nature of a CSI driver it is not really possible to simply remount everything after a restart.

**Comment by TheBeatles1994** [comment]
1. save mount info into a hostpath directory when create a pod using csi-s3 PV
2. remount using mount info when csi plugin restart

is it ok?

**Comment by lxzjd** [comment]
I have a solution，mount host's systemd to the csi container，use the mount using the systemd-run command in the container.
